### PR TITLE
Armstrong numbers

### PR DESCRIPTION
**Pull request by amymhaddad**

### CODE REVIEW

File: python/armstrong-numbers/armstrong_numbers.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import math
+
+
+def create_number_list(number):
+    number_string = ""
+    number_string += str(number) 
+
+    return [int(num) for num in number_string]
```
Comment (paulghaddad) [inline python/armstrong-numbers/armstrong_numbers.py]: So it looks like this function takes in a number such as `1234` and returns `[1, 2, 3, 4]`. You can probably just convert to a string and then split:

```
str(number).split('')
```

File: python/armstrong-numbers/armstrong_numbers.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+def is_armstrong(number):
+    number_list = create_number_list(number)
+
+    if len(number_list ) == 1:
+        return True
+    else:
+        armstrong_total = 0
+        for num in number_list:
+            armstrong_calculation = int(math.pow(num, len(number_list )))
```
Comment (paulghaddad) [inline python/armstrong-numbers/armstrong_numbers.py]: This calculation is a perfect candidate to extract to its own function. Also note the extra space after `number_list`.

Comment (paulghaddad) [reply python/armstrong-numbers/armstrong_numbers.py]: Also, do you need to use `int` here?

File: python/armstrong-numbers/armstrong_numbers.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import math
+
+
+def create_number_list(number):
+    number_string = ""
+    number_string += str(number) 
+
+    return [int(num) for num in number_string]
+
+def is_armstrong(number):
```
Comment (paulghaddad) [inline python/armstrong-numbers/armstrong_numbers.py]: This looks excellent!

**Comment by amymhaddad** [comment]
@paulghaddad Changes are made! I tried using `str(number).split('')` but that didn't give me the output I needed. It gave me a list with a single item, instead of splitting the number up.

Ex: 100 would show up as [100] and NOT [1, 0, 0], which is what I need.

File: python/armstrong-numbers/armstrong_numbers.py (hunk 1 above)
Comment (amymhaddad) [reply python/armstrong-numbers/armstrong_numbers.py]: @paulghaddad That doesn't not work b/c it returns `['153']` and I need `[1, 5, 3]`

Comment (paulghaddad) [reply python/armstrong-numbers/armstrong_numbers.py]: This is okay; I was wrong in Python 3.

**Pull request closed without merging**